File: Programming Basics with Python/06 Nested loops/More exercises/13_Prime_Pairs.py
first_pair = int(input())
second_pair = int(input())
diff_first_pair = int(input())
diff_second_pair = int(input())
for first in range(first_pair, first_pair + diff_first_pair + 1):
    first_prime = 0
    for i in range(2, first):
        if (first % i) == 0:
            break
    else:
        first_prime = first
    for second in range(second_pair, second_pair + diff_second_pair + 1):
        if first_prime == 0:
            break
        for i in range(2, second):
            if (second % i) == 0:
                break
        else:
            second_prime = second
            print(f"{first_prime}{second_prime}")

File: JS Basics/06 Nested Loops - More Exercises/13. Prime Pairs.js
function pairs(input) {
    let startFirstPair = Number(input[0]);
    let startsecondPair = Number(input[1]);
    let diffStartEndFirstPair = Number(input[2]);
    let diffStartEndSecondPair = Number(input[3]);
    let firstPrime = 0;
    let secondPrime = 0;
    let counter = 0;
    for (let firstPair = startFirstPair; firstPair <= (startFirstPair + diffStartEndFirstPair); firstPair++) {
        counter = 0;
        for (let i = 2; i <= firstPair; i++) {
            if (firstPair % i == 0) {
                counter++;
            }
        }
        if (counter == 1) {
            firstPrime = firstPair;
            for (let secondPair = startsecondPair; secondPair <= (startsecondPair + diffStartEndSecondPair); secondPair++) {
                counter = 0;
                for (let i = 2; i <= secondPair; i++) {
                    if (secondPair % i == 0) {
                        counter++;
                    }
                }
                if (counter == 1) {
                    secondPrime = secondPair;
                    console.log(`${firstPrime}${secondPrime}`);
                }
            }
        }
    }
}
pairs([10, 30, 9, 6])